src/templates/games/game_list.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block title %}
+  Games List
+{% endblock title %}
+{% block content %}
+  <h1>List of Games</h1>
+  <ul>
+    {% for game in object_list %}<li>{{ game.name }}</li>{% endfor %}
```

Comment:
This code does work after testing it by spinning up the server and adding some games in my shell. 
For future reference, it will be very good for you all to set up a fixture file with some filled out games such that you can easily test that pages like this are displaying what you want.